README.md

```diff
@@ -219,8 +219,8 @@ const openpgp = require('openpgp'); // use as CommonJS, AMD, ES6 module or via w
 
     const encrypted = await openpgp.encrypt({
         message: await openpgp.createMessage({ text: 'Hello, World!' }), // input as Message object
-        publicKeys: publicKey, // for encryption
-        privateKeys: privateKey // for signing (optional)
+        encryptionKeys: publicKey, // for encryption (public keys)
+        signingKeys: privateKey // for signing (signing keys, optional)
```

Comment:
Hey @alichry, thanks for the PR!
In my opinion, these comments become fairly redundant, now that the meaning is more clear from the code itself. So I would change this to something like:
```suggestion
        encryptionKeys: publicKey,
        signingKeys: privateKey // optional
```
And similarly for the other comments.

Reply:
Heyy @twiss 😊,

Yes, let me have a relook and remove any redundant comments :)